.github/workflows/play-store-deploy.yml

```
@@ -0,0 +1,275 @@
+      # Using the official Google Play upload action
+      # The service account JSON is passed directly from secrets (more secure than creating a file)
+      - name: Deploy to Google Play Console
+        uses: r0adkll/upload-google-play@v1.1.3
+        with:
+          serviceAccountJsonPlainText: ${{ secrets.PLAY_STORE_SERVICE_ACCOUNT_JSON }}
+          packageName: com.aryanvbw.focus
+          releaseFiles: ${{ steps.verify_build.outputs.aab_path }}
+          track: ${{ needs.determine-track.outputs.track }}
+          status: completed
+          inAppUpdatePriority: 2
+          userFraction: ${{ needs.determine-track.outputs.rollout != '100' && (needs.determine-track.outputs.rollout / 100.0) || '' }}
```

actionlint (1.7.8)

got unexpected character '/' while lexing expression, expecting 'a'..'z', 'A'..'Z', '_', '0'..'9', ''', '}', '(', ')', '[', ']', '.', '!', '<', '>', '=', '&', '|', '*', ',', ' '

(expression)